go/vt/vtgate/planbuilder/operators/rewriters.go

```diff
@@ -128,6 +128,7 @@ func FixedPointBottomUp(
 	// will loop while the rewriting changes anything
 	for ok := true; ok; ok = id != NoRewrite {
 		if DebugOperatorTree {
+			fmt.Println("Full tree:")
```

Comment:
Was this meant to be committed or just local change?

Reply:
I believe this is needed to indicate which is the full tree as we now print subparts of the tree on a rewrite.